Makefile

```diff
@@ -334,20 +334,20 @@ ci-job-clippy:
 
 define ci_setup_markdown_toc
 	$(call banner,CI-Setup: Install markdown-toc)
-	npm install -g markdown-toc
+	npm install markdown-toc
 endef
 
 .PHONY: ci-setup-markdown-toc
 ci-setup-markdown-toc:
 	$(call ci_setup_helper,\
-		command -v markdown-toc,\
-		npm install -g markdown-toc,\
+		PATH="node_modules/.bin:${PATH}" command -v markdown-toc,\
```

Comment:
I'm not an expert on the Tock make-based build system at all, but I'd try to avoid modifying $PATH wherever possible. Maybe it could be a better idea to just call the binary with the full path?

Reply:
It doesn't modify `PATH` outside of the command, just for this individual call. It's also similar to what we already do to run QEMU.

Reply:
Right. It's fine, but I'd preferably just specify the commands by their full path.

Reply:
That would require editing the `tools/toc.sh` script though, which would then break people who want to use a global install.

Reply:
Right, that should be fine then.